snapshots.go

```diff
@@ -692,128 +705,3 @@ func (m *Manager) RestoreSnapshotFromBuffer(ctx context.Context, stream string,
 	_, ss, err := m.restoreSnapshot(ctx, stream, dataReader, metadataReader, sopts)
 	return ss, err
 }
-
-// SnapshotToBuffer creates a compressed s2 backup and writes to an io.Writer
```

Comment:
`SnapshotToBufferOld` was accidentally committed during the refactor - removing.